components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -234,6 +236,7 @@ public void addOAuthApplication(OAuthAppDO consumerAppDO) throws IdentityOAuthAd
         }
     }
 
+
```

Comment:
Remove the additional line

Reply:
fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2461/commits/e89b7c23081c1c843e284226ecd37e1242f91ac0